tests/test-spawn.lua

```diff
@@ -86,13 +87,24 @@ local steps = {
                                          exit_snd = code
                                      end
                                  })
+            spawn.easy_async("ping -c1 localhost", function(stdout, stderr, reason, exit_code)
+                if stdout:match("0%% packet loss") then
+                    async_spawns_done = async_spawns_done + 1
+                end
+            end)
+            spawn.easy_async_with_shell("ping -c1 localhost | grep 'packet loss' | cut -d' ' -f6", function(stdout, stderr, reason, exit_code)
+                if stdout:match("0%%") then
+                    async_spawns_done = async_spawns_done + 1
+                end
+            end)
         end
         if spawns_done == 2 then
```

Comment:
> Is there a "base" command for local networking check?

Ping is probably installed, but don't count on it. Never write unit tests that use the network, this is, in general, a very bad practice. A test should be as atomic as possible. `echo` is as atomic a command will get without having to prepend our own directory to $PATH and use our own test commands (which would be the best solution, but takes longer to implement).